app/views/medicaid/income_job_number/edit.html.erb

```diff
@@ -10,7 +10,7 @@
       <%= f.mb_radio_set :number_of_jobs,
         '',
         [
-          { value: "1 job", label: "1 job" },
+          { value: "1 job" },
```

Comment:
answer to what this looks like with a value and no label: 
<img width="749" alt="screen shot 2017-10-12 at 11 05 57 am" src="https://user-images.githubusercontent.com/601515/31511679-584c1f3e-af3d-11e7-909f-777924ab4741.png">

Reply:
reverted at b0dbc0a